spec/linter-spec.js

```diff
@@ -21,7 +21,7 @@ describe('Flow provider for Linter', () => {
           expect(messages[0].text)
             .toEqual('number This type is incompatible with an implicitly-returned undefined.');
           expect(messages[0].filePath).toMatch(/.+constructor\.js$/);
-          expect(messages[0].trace.length).toEqual(1);
+          expect(messages[0].trace.length).toEqual(0);
```

Comment:
Since you are changing these anyway, you might as well fix this to `toBe()` (`===`) from `toEqual()` (`==`) for these simple values.

The Object equality down on lines 44, 57, and 61 should obviously stay `toEqual()` :stuck_out_tongue:.